Source/WebCore/page/FrameViewLayoutContext.h

```diff
@@ -82,6 +82,8 @@ class FrameViewLayoutContext {
     bool isInLayout() const { return layoutPhase() != LayoutPhase::OutsideLayout; }
     bool isInRenderTreeLayout() const { return layoutPhase() == LayoutPhase::InRenderTreeLayout; }
     bool inPaintableState() const { return layoutPhase() != LayoutPhase::InRenderTreeLayout && layoutPhase() != LayoutPhase::InViewSizeAdjust && (layoutPhase() != LayoutPhase::InPostLayout || inAsynchronousTasks()); }
+    bool isLayoutForced() const { return m_layoutForced; }
```

Comment:
"Force" here is too vague. Forced by what? This needs a name that clearly associates it with skipped content.

Reply:
Fair enough, I renamed to something more descriptional.